site-manage/datemanager/api/src/java/org/sakaiproject/datemanager/api/model/DateManagerUpdate.java

```diff
@@ -15,16 +15,19 @@
 
 import java.time.Instant;
 
-import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
 
-@AllArgsConstructor
+@Getter @Setter
```

Comment:
I changed this in order to insert these new or unsual values by a setter instead of having an "infinite" constructor